src/dali/transaction_resolver.py

```diff
@@ -231,70 +232,90 @@ def resolve_transactions(
     unique_id_2_transactions: Dict[AssetAndUniqueId, List[AbstractTransaction]] = {}
     transaction: AbstractTransaction
 
-    try:
-        for transaction in transactions:
-            if not isinstance(transaction, AbstractTransaction):
-                raise RP2RuntimeError(f"Internal error: Parameter 'transaction' is not a subclass of AbstractTransaction. {transaction}")
+    progress_bar = ProgressBar(
+        max_value=2 * len(transactions),
```

Comment:
Add space before and after `=` (here and below).

Also why is `max_value` twice the length of transactions? Is it to approximate `len(transactions) + len(unique_id_2_transactions)`? If so: clever!

Reply:
`black` doesn't like spaces here. I could define the arguments a few lines above, which would get spaces.

And yes - the idea was to over-estimate so the user only gets positive surprises in progress/ETA.